### DISCUSSION TITLE
i have no idea but is anyone from reddit? LOL

### DISCUSSION THREAD

**Discussion by hoazgazh** [sherlock-project/sherlock#2014] [General]
XD

**Comment by gen2pengin** [comment]
Are you referring to that angry reddit post about "not giving an EXE file"? Lol that was hilarious.

**Comment by JustNaggy** [comment]
I came after meme too LOL

**Comment by ghost** [comment]
Smelly nerd behavior

**Comment by 6DLH** [comment]
I'm sorry to learn about this project in this way
